src/bin/pgcopydb/string_utils.h

```diff
@@ -16,13 +16,22 @@
 
 /* maximum decimal int64 length with minus and NUL */
 #define INTSTRING_MAX_DIGITS 21
+#define UINT32STRING_MAX_DIGITS 10
+
 typedef struct IntString
 {
 	int64_t intValue;
 	char strValue[INTSTRING_MAX_DIGITS];
 } IntString;
 
+typedef struct UInt32String
+{
+	uint32_t intValue;
+	char strValue[INTSTRING_MAX_DIGITS];
+} UInt32String;
```

Comment:
Same comment as with the function, I believe `int64_t` can host a `int32_t` value, in a way that we don't need to add new code for that conversion...